tests/phpunit/tests/modules/wpModules.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @group modules
+ * @covers WP_Modules::get_version_query_string
```

Comment:
```suggestion
 * @coversDefaultClass WP_Modules
```

Reply:
Done as part of https://github.com/WordPress/wordpress-develop/pull/5818/commits/1aa45a6198e73869dcf4e642c3d2978888af612d.